### PR TITLE
Fix/unsmeared type i met

### PR DESCRIPTION
**Pull request by HerrHorizontal**
Fix the `unsmearedL1Jet` implementation in the `ZJetCorrectionsProducer` for the propagation of JER smearing to the TypeI-MET.

### CODE REVIEW

**Review by HerrHorizontal — COMMENTED** [review]
So @RHofsaess and I checked the new changes. Everything seems to work now as expected.

**Comment by HerrHorizontal** [comment]
@rfvc please review!

**Comment by RHofsaess** [comment]
I reviewed the code again and I think it should be all fine now.
I don't know if there is a better way for the copy, but it should be correct as it is.

**Comment by rfvc** [comment]
The changes look good to me.

File: cfg/excalibur/jec16ul/mc16_DYJets_amc/common.py [hunk 1]
```diff
@@ -78,8 +78,9 @@ def config(ch, run, jec):
     cfg['ZJetPUWeightFiles'] = [os.path.join(configtools.getPath(), 'data/pileup/mc_weights/mc16ul_DYJets_amcatnlo/PUWeights_{}_DYJetsToLL_Summer20-amcatnloFXFX_mcRun2_asymptotic_{}.root'.format(runperiod, postfix)) for runperiod in IOVS]
     cfg['ZJetPUWeightSuffixes'] = ['{}'.format(runperiod) for runperiod in IOVS]
 
-    cfg['Processors'] += ['producer:ZJetGenWeightProducer']
-    cfg['ZJetGenWeightNames'] = ['nominal', 'isrDefup', 'isrDefdown', 'fsrDefup', 'fsrDefdown']
+    # redundant?!
```
Comment (rfvc) [inline cfg/excalibur/jec16ul/mc16_DYJets_amc/common.py:81]: You are right. We should delete those lines instead of commenting them.

**Review by rfvc — APPROVED** [review]

**Pull request merged**